recipes/protobuf/all/conanfile.py

```diff
@@ -58,23 +79,32 @@ def config_options(self):
     def configure(self):
         if self.options.shared:
             self.options.rm_safe("fPIC")
+            if Version(self.version) >= "3.22.0" and is_msvc(self):
+                self.options["abseil"].shared = True
 
     def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
         if self.options.with_zlib:
             self.requires("zlib/[>=1.2.11 <2]")
+        if Version(self.version) >= "3.22.0":
```

Comment:
Please, refer where abseil become to be used.

Reply:
https://github.com/protocolbuffers/protobuf/commit/a9f1ea6371c108876649f27a5940a59cc8594768